### PR TITLE
Adding request count metrics to Perfdash 

### PR DESCRIPTION
**Pull request by krzysied**
/cc @shyamjvs

### CODE REVIEW

File: perfdash/parser.go [hunk 1]
```diff
@@ -29,6 +30,22 @@ func stripCount(data *perftype.DataItem) {
...
+	requestCountString, ok := data.Labels["Count"]
+	if !ok {
+		return fmt.Errorf("no \"Count\" label")
+	}
+        requestCount, err := strconv.ParseFloat(requestCountString, 64)
+	if err != nil {
+		return fmt.Errorf("couldn't parse count: %v", err)
+	}
+	data.Data["RequestCount"] = requestCount
+
```
Comment (shyamjvs) [inline perfdash/parser.go]: Remove unnecessary blank line.

File: perfdash/parser.go [hunk 2]
```diff
@@ -29,6 +30,22 @@ func stripCount(data *perftype.DataItem) {
...
 }
 
+func createRequestCountData(data *perftype.DataItem) error {
+	data.Unit = ""
+	data.Data = make(map[string]float64)
+	requestCountString, ok := data.Labels["Count"]
+	if !ok {
+		return fmt.Errorf("no \"Count\" label")
+	}
+        requestCount, err := strconv.ParseFloat(requestCountString, 64)
```
Comment (shyamjvs) [inline perfdash/parser.go]: Remove extra space at beginning of line.

File: perfdash/parser.go [hunk 3]
```diff
@@ -29,6 +30,22 @@ func stripCount(data *perftype.DataItem) {
 	delete(data.Labels, "Count")
 }
 
+func createRequestCountData(data *perftype.DataItem) error {
+	data.Unit = ""
+	data.Data = make(map[string]float64)
+	requestCountString, ok := data.Labels["Count"]
+	if !ok {
+		return fmt.Errorf("no \"Count\" label")
```
Comment (shyamjvs) [inline perfdash/parser.go]: Maybe `Couldn't find 'Count' label`?

File: perfdash/parser.go [hunk 4]
```diff
@@ -99,3 +116,24 @@ func parseResourceUsageData(data []byte, buildNumber int, testResult *BuildData)
...
+		fmt.Fprintf(os.Stderr, "error parsing JSON in build %d: %v %s\n", buildNumber, err, string(data))
+		return
+	}
+	if testResult.Version == "" {
+		testResult.Version = obj.Version
+	}
+	if testResult.Version == obj.Version {
+		for i := range obj.DataItems {
+			if err := createRequestCountData(&obj.DataItems[i]); err != nil {
+				fmt.Fprintf(os.Stderr, "error creating request count data in build %d dataItem %d: %v\n", buildNumber, i, err)
```
Comment (shyamjvs) [inline perfdash/parser.go]: You need to "continue" after this.

Comment (krzysied) [reply perfdash/parser.go]: "continue" added.

**Comment by shyamjvs** [comment]
/lgtm

Thanks!

**Pull request merged**